attributes/default.rb

```diff
@@ -4,7 +4,6 @@
 default["mimic"]["path"] = "/opt/mimic"
 default["mimic"]["virtualenv"] = "/srv/mimic"
 
-default["mimic"]["github"] = "https://github.com/rackerlabs/mimic.git"
-default["mimic"]["revision"] = nil
+default["mimic"]["version"] = nil
```

Comment:
Currently mimic is on version 2.0.0, according to the source. Not sure if that is synced in pip, but we should probably enforce a version by default or 3.x will break us.

https://github.com/rackerlabs/mimic/blob/master/setup.py#L16